circle.yml

```diff
@@ -1,8 +1,20 @@
 machine:
   node:
     version: 6.0.0
+  pre:
+    - mkdir -p ~/.yarn-cache
 
 dependencies:
+  pre:
+    - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
My understanding is that yarn is already installed these days.

I'm going to upgrade the instance to Ubuntu 14 as part of this so we should be able to remove this step.